src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurRequestExtractFileValidationServiceImpl.java

```diff
@@ -184,22 +182,29 @@ private boolean employeeIdIsValid(ConcurRequestExtractRequestDetailFileLine deta
             return false;
         }
         else {
-            try {
-                Person employee = getPersonService().getPersonByEmployeeId(detailFileLine.getEmployeeId());
-                if (ObjectUtils.isNotNull(employee)) {
-                    return true;
-                }
-                else {
-                    detailFileLine.getValidationResult().addMessage(getConfigurationService().getPropertyValueAsString(ConcurKeyConstants.CONCUR_REQUEST_EXTRACT_EMPLOYEE_ID_NOT_FOUND_IN_KFS));
-                    return false;
-                }
-            }
-            catch (Exception e) {
-                detailFileLine.getValidationResult().addMessage(getConfigurationService().getPropertyValueAsString(ConcurKeyConstants.CONCUR_REQUEST_EXTRACT_EMPLOYEE_ID_NOT_FOUND_IN_KFS));
+            boolean validPerson = getConcurPersonValidationService().validPerson(detailFileLine.getEmployeeId());
```

Comment:
Should this be: if !validPerson add the error message and then return validPerson?